PowerUp/app/src/main/java/powerup/systers/com/IntroActivity6.java

```diff
@@ -0,0 +1,17 @@
+package powerup.systers.com;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+public class IntroActivity6 extends Fragment {
```

Comment:
Please use coherent names. If something is a Fragment, it should be called Fragment so it's not misleading